bot/exts/recruitment/talentpool/_review.py

```diff
@@ -120,6 +130,77 @@ async def make_review(self, user_id: int) -> typing.Tuple[str, Optional[Emoji]]:
         review = "\n\n".join((opening, current_nominations, review_body, vote_request))
         return review, seen_emoji
 
+    async def archive_vote(self, message: PartialMessage, passed: bool) -> None:
+        """Archive this vote to #nomination-archive."""
+        message = await message.fetch()
+
+        # We consider that if a message has been sent less than 2 second before the one being archived
+        # it is part of the same nomination.
+        # For that we try to get messages sent in this timeframe until none is returned and NoMoreItems is raised.
+        messages = [message]
+        with contextlib.suppress(NoMoreItems):
+            async for new_message in message.channel.history(before=message.created_at):
+                if messages[-1].created_at - new_message.created_at > timedelta(seconds=2):
+                    break
+                messages.append(new_message)
+
+        parts = []
+        for message_ in messages[::-1]:
+            parts.append(message_.content)
+            parts.append("\n" if message_.content.endswith(".") else " ")
+        content = "".join(parts)
+
+        # We assume that the first user mentioned is the user that we are voting on
+        user_id = int(MENTION_RE.search(content).group(1))
+
+        # Get reaction counts
+        seen = await count_unique_users_reaction(
+            messages[0],
+            lambda r: "ducky" in str(r) or str(r) == "\N{EYES}",
+            count_bots=False
+        )
+        upvotes = await count_unique_users_reaction(
+            messages[0],
+            lambda r: str(r) == "\N{THUMBS UP SIGN}",
+            count_bots=False
+        )
+        downvotes = await count_unique_users_reaction(
+            messages[0],
+            lambda r: str(r) == "\N{THUMBS DOWN SIGN}",
+            count_bots=False
+        )
+
+        # Remove the first and last paragraphs
+        stripped_content = content.split("\n\n", maxsplit=1)[1].rsplit("\n\n", maxsplit=1)[0]
+
+        result = f"**Passed** {Emojis.incident_actioned}" if passed else f"**Rejected** {Emojis.incident_unactioned}"
+        colour = Colours.soft_green if passed else Colours.soft_red
+        timestamp = datetime.utcnow().strftime("%d/%m/%Y")
```

Comment:
Would y-m-d be better here to avoid confusion? The current standard in the archive is to use d-m-y, but that's just arbitrary.

Reply:
I just followed what was already being used, but I don't mind changing it.

Reply:
Changed in b5f5668